types/pouchdb-core/index.d.ts

```diff
@@ -55,6 +55,15 @@ declare namespace PouchDB {
             rev: RevisionId;
             status: Availability;
         }
+        interface RevisionDiffList {
+            [documentId : string]: string[];
+        }
+        interface RevisionDiff {
+            [change : string]: string[];
```

Comment:
At least we can do
```ts
interface RevisionDiff {
    missing?: string[];
    possible_ancestors?: string[];
}
```
if we indeed know those are the possible outcomes. It's not awesome, but it works.